diesel/src/migrations/migration_error.rs

```diff
@@ -55,6 +55,7 @@ impl From<io::Error> for MigrationError {
 }
 
 #[derive(Debug, PartialEq)]
+#[cfg_attr(feature = "dev", allow(enum_variant_names))]
```

Comment:
By which I mean I don't feel strongly about this enum, and I'd be fine with changing the names for this one.